packages/component-library-react/README.md

```diff
@@ -1,5 +1,37 @@
 <!-- @license CC0-1.0 -->
 
-# React Components
+# Component library for React apps
 
-The `@utrecht/component-library-react` package contains React implementations of various components.
+The `@utrecht/component-library-react` package contains React implementations of various components. You can use this package in React apps, both client side and server side, to render the right HTML elements with the design system class names.
+
+Combine this with other packages for the full experience: you also need to add the package with CSS to style these components, as well as the package with design tokens for your theme.
```

Comment:
I think this could have even more explanation maybe something like this?
> As the components use the CSS components, you will also need to add the package with the CSS implementations. For ensuring the components are styled with your theme, don't forget to include the package with your design tokens for these components.